docs/features/pixi_global_manifest.md

```diff
@@ -0,0 +1,222 @@
+The syntax for `MAPPING` is `exposed_name=binary_name`, so for example `python3.10=python`.
+```
+pixi expose --environment ENV --package PACKAGE [MAPPING]...
+```
+
+Ensure that the environments on the machine reflect the state in the manifest.
+The manifest is the single source of truth.
+Only if there's no manifest, will the data from existing environments be used to create a manifest.
+`pixi global sync` is implied by most other `pixi global` commands.
+```
+pixi global sync
+```
+
+List all environments, their specs and exposed binaries
+```
+pixi global list
+```
+
+
+### Simple workflow
+
+Create environment `python`, install package `python=3.10.*` and expose all binaries of that package
+```
+pixi global install python=3.10.*
+```
+
+Upgrade all packages in environment `python`
+```
+pixi global upgrade python
+```
+
+Remove environment `python`
+```
+pixi global uninstall python
+```
+
+Create environment `python` and `pip`, install corresponding packages and expose all binaries of that packages
+```
+pixi global install python pip
+```
+
+Remove environments `python` and `pip`
+```
+pixi global uninstall python pip
+```
+
+### Injecting dependencies
+
+Create environment `python`, install package `python` and expose all binaries of that package.
+Then add package `hypercorn` to environment `python` but doesn't expose its binaries.
+
+```
+pixi global install python
+pixi global inject --environment=python hypercorn
+```
+
+Update package `cryptography` (a dependency of `hypercorn`) in environment `python`
+
+```
+pixi update --environment python cryptography
+```
+
+Then remove `hypercorn` again.
+```
+pixi global remove --environment=python hypercorn
+```
+
+
+### Specifying which binaries to expose
+
+Make a new environment `python_3_10` with package `python=3.10` and no exposed binaries
+```
+pixi global install --environment python_3_10 --expose "python3.10=python" python=3.10
+```
+
+Now `python3.10` is available.
+
+
+Run the following in order to expose `python` from environment `python_3_10` as `python310` instead.
+
+```
+pixi global expose --environment python_3_10 --package python "python310=python"
+```
+
+Now `python310` is available, but `python3.10` isn't anymore.
+
+!!! note
+
+    It should be possible to infer `--package`, let's discuss if there are edge cases to consider.
+
+### Syncing
+
+Most `pixi global` sub commands imply a `pixi global sync`.
+
+- Users should be able to change the manifest by hand (creating or modifying (adding or removing))
+- Users should be able to "export" their existing environments into the manifest, if non-existing.
+- The manifest is always "in sync" after `install`/`remove`/`inject`/`other global command`.
+
+
+First time, clean computer.
+Running the following creates manifest and `~/.pixi/envs/python`.
+```
+pixi global install python
+```
+
+Delete `~/.pixi` and syncing, should add environment `python` again as described in the manifest
+```
+rm `~/.pixi`
```

Comment:
Wont this also delete the manifest?

Reply:
Yes, my bad.
This should be `~/.pixi/envs`.
Fixed!